src/core/connection.c

```diff
@@ -435,10 +439,17 @@ QuicConnApplySettings(
 
     if (Settings->ServerResumptionLevel > QUIC_SERVER_NO_RESUME) {
         QUIC_DBG_ASSERT(!Connection->State.Started);
-        //
-        // TODO: allocate memory for handshake TP here
-        //
-        Connection->State.ResumptionEnabled = TRUE;
+        Connection->HandshakeTP = QUIC_ALLOC_NONPAGED(sizeof(*Connection->HandshakeTP));
```

Comment:
A later PR will use a pool for these allocations.

Reply:
Put a TODO please so it's not forgotten.